Il2CppInspector.Common/Reflection/TypeInfo.cs

```diff
@@ -86,7 +91,7 @@ private class TypeArgumentsComparer : EqualityComparer<TypeInfo[]>
         public bool ContainsGenericParameters => (HasElementType && ElementType.ContainsGenericParameters) || IsGenericParameter || genericArguments.Any(ga => ga.ContainsGenericParameters);
 
         // Custom attributes for this member
-        public override IEnumerable<CustomAttributeData> CustomAttributes => CustomAttributeData.GetCustomAttributes(this);
+        public override IEnumerable<CustomAttributeData> CustomAttributes => CustomAttributeData.GetCustomAttributes(genericTypeDefinition ?? this);
```

Comment:
Could you explain this one to me please? 🙂

Reply:
Custom attributes need to be looked up using the definition (since it requires `TypeInfo.Definition`), and cannot reference type parameters (AFAICT), so they can't be generically substituted anyway.

Reply:
So if it's any kind of open or closed generic type it fetches from the generic type definition, otherwise it's just treated as normal?

Reply:
Yeah, concretized generics use the same attributes as the definition.